x-pack/legacy/plugins/watcher/public/np_ready/application/sections/watch_edit/components/threshold_watch_edit/action_fields/webhook_action_fields.tsx

```diff
@@ -29,13 +29,15 @@ interface Props {
 
 const HTTP_VERBS = ['head', 'get', 'post', 'put', 'delete'];
 
+const SCHEME_VERBS = ['', 'http', 'https'];
```

Comment:
I don't think it's necessary to have an empty first option. I believe `http` is the default value in ES, so I think it makes sense to have that selected by default in the UI.

Reply:
nit: I might suggest renaming this constant. I wouldn't consider `http` and `https` as verbs.

Reply:
> I don't think it's necessary to have an empty first option. I believe `http` is the default value in ES, so I think it makes sense to have that selected by default in the UI.

I left an empty first option because when I set `const SCHEME_VERBS = ['http', 'https'];` and click on show request the scheme field doesn't show up on the generated request. It looks like it's because the EUI FieldText is set to update onChange. I can set `const SCHEME_VERBS = ['http', 'https'];` , but will have to add more code to deal with that.

> nit: I might suggest renaming this constant. I wouldn't consider `http` and `https` as verbs.

No problem, I will sent it as `const SCHEME = ['http', 'https'];`

Reply:
You should be able to provide a default value to `scheme` in the `webhook_action.js` model file.

For example:
```
this.scheme = get(props, 'scheme', 'http');
```

Reply:
@alisonelizabeth thanks for the advice, i've uploaded the changes